khal/ui/__init__.py

```diff
@@ -870,7 +869,7 @@ def new(self, date, end=None):
             self.pane.window.alert(('alert', 'No writable calendar.'))
             return
         if end is None:
-            start = dt.datetime.combine(dt.date, dt.time(dt.datetime.now().hour))
+            start = dt.datetime.combine(dt.date.today (), dt.time(dt.datetime.now().hour))
```

Comment:
This should actually be `dt.datetime.combine(date, dt.time(dt.datetime.now().hour())`, date is a variable given to `new()` here.